openassessment/templates/openassessmentblock/staff/oa_staff_incomplete_waiting.html

```diff
@@ -0,0 +1,31 @@
+{% load i18n %}
+
+<li id="openassessment__staff-assessment" class="openassessment__steps__step step--staff-assessment">
+    <header class="step__header">
```

Comment:
Yeah, I followed the existing pattern of creating a separate for each possible state, but in reality it is easy enough to use a single template with context variables. I'm pushing up that change.